track.go

```diff
@@ -171,6 +172,10 @@ func (track *baseTrack) bind(ctx webrtc.TrackLocalContext, specializedTrack Trac
 	for _, wantedCodec := range ctx.CodecParameters() {
 		logger.Debugf("trying to build %s rtp reader", wantedCodec.MimeType)
 		encodedReader, err = specializedTrack.NewRTPReader(wantedCodec.MimeType, uint32(ctx.SSRC()), rtpOutboundMTU)
+		if track.err != nil {
+			err = track.err
```

Comment:
I think this block also need `track.errMu.Lock()`

Reply:
Thanks for noticing! I added the lock so It should be good now.